draft-ietf-tsvwg-rfc4960-bis.xml

```diff
@@ -708,7 +708,7 @@ have been incorporated in this document:</t>
 <li><t>Improve the terminology to make clear that this specification does not
        describe a full mesh architecture.</t></li>
 <li><t>Improve the description of sequence number generation
-       (TSN and SSN).</t></li>
+       (TSN and Stream Sequence Number).</t></li>
```

Comment:
Using `(Transmission Sequence Number and Stream Sequence Number)` for consistency.

Reply:
Sure.  I just noticed that this was the only instance of "TSN" in the doc and so expanded it like the other occurrences.  (I thought I had seen "SSN" all over the place but seem to have been wrong about that.)